src/Validation/Validator.php

```diff
@@ -12,6 +13,10 @@ final class Validator implements \Lcobucci\JWT\Validator
      */
     public function assert(Token $token, Constraint ...$constraints): void
     {
+        if (count($constraints) === 0) {
```

Comment:
I know this is kind of micro optimisation, but let's avoid using `count()` here.
We can do `if ($constraints === [])`.